src/services/user.service.ts

```diff
@@ -53,10 +36,10 @@ export class UserService {
   public async createUserAndReturnToken(
     email: string,
     password: string
-  ): Promise<string> {
+  ): Promise<Object> {
```

Comment:
Don't use Object. Define interface. Also don't return token on creating the user